docs/getting-started/installation.md

```diff
@@ -17,37 +18,162 @@ This can be done with the following command:
     
 `/usr/sbin/softwareupdate --install-rosetta --agree-to-license`
 
-## Virtual Environment (Optional)
+
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.box(
+        rx.alert_title("Error ", rx.code("command not found: reflex"),),
+        rx.alert_description(
+            "If you install Reflex with no virtual environment and get this error it means your PATH cannot find the reflex package. A virtual environment should solve this problem.",
+        ),
+    ),
+    status="warning",
+)
+```
+
+
+## Virtual Environment (Recommended)
 
 We recommend creating a virtual environment for your project.
 
 Below are some tools you can use to manage environments:
 
 - [venv]({constants.VENV_URL})
 - [poetry]({constants.POETRY_URL})
+- [conda]({constants.CONDA_URL})
+
+```python eval
+rx.box(height=8)
+```
+
+
+
+## Installing on macOS/Linux
 
-## Installing
+Reflex's recommended package manager and environment manager for macOS and Linux are [pip]({constants.PIP_URL}) and [venv]({constants.VENV_URL}), respectively. `venv` is a part of [The Python Standard Library]({constants.PYTHON_STANDARD_LIBRARY}) and comes bundled with your installation of Python. See instructions on how to install and use `pip` below.
 
-Reflex is available as a [pip](constants.PIP_URL) package.
+
+### Install pip
+
+Install `pip`. More details about installing pip can be found in [pip's documentation]({constants.PIP_DOCS}).
+
+On a macOS:
 
 ```bash
-pip install reflex
+python -m ensurepip --upgrade
```

Comment:
if we recommend using a venv then i don't think this is necessary, because the venv already comes with pip.

having users install a "system pip" is dangerous and can lead them to making mistakes like `pip install` outside of a virtualenv and trashing their user-local or (worse) system package cache

Reply:
got rid of all of this